dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/config/FloridaConfig.java

```diff
@@ -304,7 +323,7 @@
      */
     @DefaultValue("4")
     @PropertyName(name = "dyno.ardb.rocksdb.minwritebuffernametomerge")
-    public int getArdbRocksDBMinWriteBuffersToMerge();
+    public int getRocksDBMinWriteBuffersToMerge();
```

Comment:
why remove ardb from the function name?

Reply:
Because this is a RocksDB configuration. I am trying to differentiate between RocksDB and ARDB configurations.